NewHaven/src/VGMap.cpp

```diff
@@ -411,4 +422,47 @@ string VGMap::castResourceTypesToString(ResourceTypes type){
 int VGMap::getNumUnplayed(){
     // essentially we loop through
     return (*SIZE - *playCounter);
+}
+
+string VGMap::getBoardString() const {
+    ostringstream vBoard;
+    const string spacer = "    ";
+    const string inner_spacer ="  ";
+    vBoard << spacer << spacer << spacer << spacer << spacer << "***" << *name << "***" << spacer << spacer << '\n';
+    vBoard << "------------" << "-------------------------------------------------------------" << '\n';
+    int num_row{0};
```

Comment:
Same thing I said about the `getBoardString()` of `GBMap`, this can simplify `PrintGraph()`.